### PR TITLE
[ISSUE_TEMPLATE] First implementation of the issue template

### PR DESCRIPTION
**Pull request by C47D**
Hi,

@kisvegabor, could you help us to get this file as an issue template for this repo? I tried to follow the issue template of the main lvgl repo.

Let me know if you have any suggestions.

Regards

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue-template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Issue template
```
Comment (kisvegabor) [inline .github/ISSUE_TEMPLATE/issue-template.md]: use it only for bug reports? 
If so then maybe a title like "Bug reports" should be better,

File: .github/ISSUE_TEMPLATE/issue-template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+**Development kit used**
+
+**Development machine OS**
+
+**Compilation warnings/errors (if available)**
+
+**If possible, copy the compilation log into a file and attach it here**
+
+
+We use GitHub issues for development related discussions.
```
Comment (kisvegabor) [inline .github/ISSUE_TEMPLATE/issue-template.md]: I'd move it to the top to increase the chance that people will read it.

File: .github/ISSUE_TEMPLATE/issue-template.md (hunk 1 above)
Comment (C47D) [reply .github/ISSUE_TEMPLATE/issue-template.md]: Done in 7e5deae

File: .github/ISSUE_TEMPLATE/issue-template.md (hunk 2 above)
Comment (C47D) [reply .github/ISSUE_TEMPLATE/issue-template.md]: Done in 7e5deae

**Comment by kisvegabor** [comment]
Merged!

It looks good in the practice too.

**Comment by C47D** [comment]
Great, thanks for the help!

**Pull request merged**